src/components/data-management/LaunchAnalysisButton.jsx

```diff
@@ -130,7 +132,7 @@ const LaunchAnalysisButton = () => {
     if (!canLaunchAnalysis()) {
       return (
         <Tooltip
-          title='Ensure all samples are uploaded and all metadata are inserted'
+          title='Ensure that all samples are uploaded successfully and all relevant metadata inserted.'
```

Comment:
and all relevant metadata **is** inserted

Reply:
Thanks for catching that :)